data/json/recipes/recipe_weapon.json

```diff
@@ -336,6 +336,70 @@
       [ [ "cordage", 2, "LIST" ], [ "filament", 200, "LIST" ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "compositebow",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "fabrication",
+    "skills_required": [ [ "archery", 2 ] ],
+    "difficulty": 4,
+    "time": 15000,
+    "autolearn": true,
+    "book_learn": [ [ "recipe_bows", 4 ], [ "manual_archery", 5 ] ],
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [
+      [ [ "stick", 3 ], [ "2x4", 2 ] ],
+      [ [ "bone", 3 ], [ "bone_human", 3 ] ],
+      [ [ "cordage", 2, "LIST" ], [ "filament", 200, "LIST" ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "recurbow",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "fabrication",
+    "skills_required": [ [ "mechanics", 2 ] ],
+    "difficulty": 5,
+    "time": 30000,
+    "book_learn": [ [ "recipe_bows", 6 ] ],
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "HAMMER", "level": 2 }, { "id": "SAW_M_FINE", "level": 1 } ],
+    "components": [
+      [ [ "stick", 3 ], [ "2x4", 2 ] ],
+      [ [ "steel_chunk", 1 ], [ "scrap", 3 ] ],
+      [ [ "plastic_chunk", 2 ] ],
+      [ [ "duct_tape", 100 ], [ "superglue", 1 ] ],
```

Comment:
We have `adhesive` requirement for this, but I'm not sure if bone glue if sturdy enough.

Reply:
I agree, I don't think bone glue would be strong enough, and i'd worry about it falling apart from use.